### DISCUSSION TITLE
Installazione Home Assistant OS - errore URL

### DISCUSSION THREAD

**Discussion by Paol-O** [indomus/forum#3179] [GENERALE - Discussioni generiche]
Salve a tutti
sto provando ad installare Home Assistant OS su mini PC, seguendo la guida.
nel momento in cui inserisco la riga di comando **curl -L 'https://indomus.it/link/home-assistant-os-x86-64' --output /var/crash/hassos.iso.xz** il sistema, dopo aver completato la percentuale del 100% mi risponde con:

**curl: (3) URL reject: No host part in the URL**

per cui non riesco a proseguire oltre.

Nel link della riga di comando ho già notato il doppio "-" davanti output che generava un altro errore, e l'ho sostituito con un solo "-", ma ottengo l'errore indicato sopra

**Comment by snakuzzo** [comment]
Quindi mettiamo i trattini a sensazione?
Se fai...

```
curl --help
```

vedrai che i due trattini sono corretti se usi l'opzione non abbreviata. Perché o digiti `--output` o `-o`

Ad ogni modo il comando è corretto (appena testato), avrai sbagliato qualcosa tu col copia-incolla

**Reply by Paol-O** [reply]
Ciao, grazie per la tua risposta. 
Ho provato a riscrivere e a copia/incollare il comando, ma l'operazione non viene mai completata, tirando un errore (nel caso dei 2 - l'errore arriva prima, per questo avevo immaginato fossero sbagliati) 

Non so cosa possa non andare nella mia installazione, per questo ho chiesto aiuto

**Comment by KC-inDomus** [comment]
> Salve a tutti sto provando ad installare Home Assistant OS su mini PC, seguendo la guida. nel momento in cui inserisco la riga di comando **curl -L 'https://indomus.it/link/home-assistant-os-x86-64' --output /var/crash/hassos.iso.xz** il sistema, dopo aver completato la percentuale del 100% mi risponde con:
> 
> **curl: (3) URL reject: No host part in the URL**
> 
> per cui non riesco a proseguire oltre.
> 
> Nel link della riga di comando ho già notato il doppio "-" davanti output che generava un altro errore, e l'ho sostituito con un solo "-", ma ottengo l'errore indicato sopra

sicuro di aver usato i caratteri corretti, in primis gli apici corretti?

**Reply by Paol-O** [reply]
Penso di sì, anche perché in successivi tentativo ho copiato e incollato il testo direttamente preso dalla guida, anziché riscriverlo

**Reply by KC-inDomus** [reply]
se l'host è connesso a internet e usi quel comando, funziona al 101%. verifica bene il tutto, stai commettendo tu un errore.

**Comment by Paol-O** [comment]
![PXL_20240816_141641587f](https://github.com/user-attachments/assets/843bb9e3-36e6-4aa5-a768-410d877b08a0)

questa la situazione a schermo

**Reply by snakuzzo** [reply]
Cambia la directory di output, non usare quella indicata. Usa la home del tuo utente...

```
--output ~/hassos.iso.xz
```

**Reply by indomus** [reply]
Ciao, la guida è stata rivista e corretta. La nuova versione di Ubuntu 24 cambia i diritti sul filesystem, nello specifico quelli relativi a /var/crash.
Ribattila, ora funziona tutto.
